/**:
  ros__parameters:
    platform_height: 2
    wheel_radius: 0.55